backend/src/main/java/dccs/academy/seeders/DepartmentSeed.java

```diff
@@ -0,0 +1,44 @@
+package dccs.academy.seeders;
+
+import dccs.academy.dtos.DepartmentDto;
+import dccs.academy.repositories.DepartmentRepository;
+import dccs.academy.services.DepartmentService;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.event.Observes;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+import io.quarkus.runtime.StartupEvent;
+
+import java.util.HashSet;
+
+@ApplicationScoped
+public class DepartmentSeed {
+
+    @Inject
+    DepartmentService departmentService;
+
+    @Inject
+    DepartmentRepository departmentRepository;
+
+    @Transactional
+    void onStart(@Observes StartupEvent ev) {
+        seedDepartments();
+    }
+
+    public void seedDepartments() {
+        var departments = new HashSet<String>();
+        departments.add("HR");
+        departments.add("Engineering");
+        departments.add("Sales");
+        departments.add("Marketing");
+        departments.add("Finance");
+
+        if (departmentRepository.listAll().isEmpty()) {
```

Comment:
This is bad practice. Content of department table can grow huge and you listAll of entries here just to check if it is empty. Please make a query to check if table is empty.